Dockerfile

```diff
@@ -8,25 +8,87 @@ RUN apt-get update \
         patch pkg-config swig unzip wget xz-utils \
         autoconf autotools-dev automake texinfo dejagnu \
         build-essential prelink autoconf libtool libltdl-dev \
-        # testing packages: libgconf-2-4 is necessary for running chromium
-        libgconf-2-4 "chromium=90.*" \
+        gnupg2 libdbus-glib-1-2 \
   && rm -rf /var/lib/apt/lists/*
 
 ADD docs/requirements-doc.txt requirements.txt /
 
 RUN pip3 --no-cache-dir install -r /requirements.txt \
   && pip3 --no-cache-dir install -r /requirements-doc.txt
 
-# Get firefox 70.0.1 and geckodriver
-RUN wget -qO- https://ftp.mozilla.org/pub/firefox/releases/87.0/linux-x86_64/en-US/firefox-87.0.tar.bz2 | tar jx \
-  && ln -s $PWD/firefox/firefox /usr/local/bin/firefox \
-  && wget -qO- https://github.com/mozilla/geckodriver/releases/download/v0.29.1/geckodriver-v0.29.1-linux64.tar.gz | tar zxC /usr/local/bin/
+# Get Chrome and Firefox (borrowed from https://github.com/SeleniumHQ/docker-selenium)
 
-# Get recent version of chromedriver
-RUN wget --quiet https://chromedriver.storage.googleapis.com/90.0.4430.24/chromedriver_linux64.zip \
-  && unzip chromedriver_linux64.zip \
-  && mv $PWD/chromedriver /usr/local/bin \
-  && rm -f chromedriver_linux64.zip
+ARG CHROME_VERSION="google-chrome-stable"
+ARG CHROME_DRIVER_VERSION
+ARG FIREFOX_VERSION="latest"
+# Note: default geckodriver version needs to be updated manually
+ARG GECKODRIVER_VERSION="0.30.0"
```

Comment:
Related to the previous comment, maybe let's move them to ENV ? That way they are still defined separately but users don't need to pass 4 args if they want to rebuild the docker image.

Reply:
I guess ARG with a proper default value would be fine. Users don't have to pass args if they don't want to specify the browser version they want to use.